openrank-sdk/src/lib.rs

```diff
@@ -124,12 +127,362 @@ pub enum SdkError {
+    /// 2. Calls the Sequencer to get the `TrustUpdate`s.
+    pub async fn get_trust_updates(
+        &self, from: Option<String>, size: Option<usize>,
+    ) -> Result<Vec<TrustUpdate>, SdkError> {
+        let from = if let Some(data) = from {
+            let tx_hash_bytes = hex::decode(data).map_err(SdkError::HexError)?;
+            let trust_update_tx_hash = TxHash::from_bytes(tx_hash_bytes);
+            Some(trust_update_tx_hash)
+        } else {
+            None
+        };
+        let client = HttpClient::builder()
+            .build(self.config.sequencer.endpoint.as_str())
+            .map_err(SdkError::JsonRpcClientError)?;
+        let results_query = GetTrustUpdateQuery::new(from, size);
+        let trust_updates: Vec<TrustUpdate> = client
+            .request("sequencer_get_trust_updates", vec![results_query])
+            .await
+            .map_err(SdkError::JsonRpcClientError)?;
+        Ok(trust_updates)
+    }
+
+    /// 1. Creates a new `Client`, which can be used to call the Sequencer.
+    /// 2. Calls the Sequencer to get the `SeedUpdate`s.
+    pub async fn get_seed_updates(
+        &self, from: Option<String>, size: Option<usize>,
+    ) -> Result<Vec<SeedUpdate>, SdkError> {
+        let from = if let Some(data) = from {
+            let tx_hash_bytes = hex::decode(data).map_err(SdkError::HexError)?;
+            let seed_update_tx_hash = TxHash::from_bytes(tx_hash_bytes);
+            Some(seed_update_tx_hash)
+        } else {
+            None
+        };
+
+        let client = HttpClient::builder()
+            .build(self.config.sequencer.endpoint.as_str())
+            .map_err(SdkError::JsonRpcClientError)?;
+        let results_query = GetSeedUpdateQuery::new(from, size);
+        let seed_updates: Vec<SeedUpdate> = client
+            .request("sequencer_get_seed_updates", vec![results_query])
+            .await
+            .map_err(SdkError::JsonRpcClientError)?;
+        Ok(seed_updates)
+    }
+
+    pub fn check_score_integrity(
+        votes: Vec<bool>, computed_scores: Vec<ScoreEntry>, correct_scores: Vec<ScoreEntry>,
+    ) -> Result<bool, SdkError> {
+        let mut computed_scores_map = HashMap::new();
+        for score in computed_scores {
+            computed_scores_map.insert(score.id().clone(), *score.value());
+        }
+
+        let mut correct_scores_map = HashMap::new();
+        for score in correct_scores {
+            correct_scores_map.insert(score.id().clone(), *score.value());
+        }
+
+        let is_converged = is_converged_org(&computed_scores_map, &correct_scores_map);
+        let votes = votes.iter().fold(true, |acc, vote| acc & vote);
+
+        Ok(is_converged & votes)
+    }
+}
+
 /// 1. Reads a CSV file and get a list of `TrustEntry`.
-/// 2. Creates a new `Client`, which can be used to call the Sequencer.
-/// 3. Sends the list of `TrustEntry` to the Sequencer.
+/// 2. Call the `OpenRankSDK::trust_update` func
 pub async fn update_trust(
```

Comment:
All of these helper functions, like `update_trust`, `update_seed` etc. should be moved back to `main.rs` since they are helper functions for the CLI and work with filesystem directly

Reply:
Done https://github.com/openrankprotocol/openrank/pull/218/commits/20a2dc1cf9b5732cc6795fa4b8fa1b2627bb1624